### PR TITLE
fix(topology): scrollable detail sidebar

### PR DESCRIPTION
**Pull request by tthvo**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] Signed the last commit: `git commit --amend --signoff`
_______________________________________________

Related to #891 

## Description of the change:

Add some css rules to allow detail content to be scrollable. This prevents the topology view from stretching the visualization surface, causing control bar to be pushed down.

## Motivation for the change:

Nicer UX and closer to OCP design. But there is one difference. For OCP, the whole sidebar is scrollable, including headers and close button. For cryostat, only the sidebar content body is scrollable (all below the tab bar).

## Screenshots

Screenshot from 2023-04-03 22-06-45

### CODE REVIEW

**Comment by andrewazores** [comment]
Seems like the content container doesn't extend all the way to the bottom of the parent component?

image

1920x1200 display, maybe this is hardcoded for 1080p?

**Comment by tthvo** [comment]
Yehh I am using `height: 36.2em;` to constraint the height of content body. Been trying to substitute with `vh` but there is some weird rendering with Match Expression Visualizer. Let me try again...

**Comment by tthvo** [comment]
Phew finally, its really a flex box :)) The scrolling no longer needs to have fixed height anymore. Just some flex rules should do the tricks. How about now?

**Review by andrewazores — APPROVED** [review]
